lib/omnibus/generator_files/omnibus.rb.erb

```diff
@@ -40,6 +40,7 @@
 # solaris_compiler 'gcc'
 # build_retries 5
 # fetcher_read_timeout 120
+# fetcher_retries 120
```

Comment:
Probably just a copy/pasta error but we want a slightly smaller default here. :)